[2024-04-13T18:50:56.611+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fruits_data_pipeline.create_fruits_table manual__2024-04-13T18:50:49.764352+00:00 [queued]>
[2024-04-13T18:50:56.639+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fruits_data_pipeline.create_fruits_table manual__2024-04-13T18:50:49.764352+00:00 [queued]>
[2024-04-13T18:50:56.644+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-13T18:50:56.680+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): create_fruits_table> on 2024-04-13 18:50:49.764352+00:00
[2024-04-13T18:50:56.693+0000] {standard_task_runner.py:60} INFO - Started process 588 to run task
[2024-04-13T18:50:56.700+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fruits_data_pipeline', 'create_fruits_table', 'manual__2024-04-13T18:50:49.764352+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/eksplorasi.py', '--cfg-path', '/tmp/tmp6mkgzrc0']
[2024-04-13T18:50:56.704+0000] {standard_task_runner.py:88} INFO - Job 174: Subtask create_fruits_table
[2024-04-13T18:50:56.814+0000] {task_command.py:423} INFO - Running <TaskInstance: fruits_data_pipeline.create_fruits_table manual__2024-04-13T18:50:49.764352+00:00 [running]> on host b46e24c16c09
[2024-04-13T18:50:56.958+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dzikri' AIRFLOW_CTX_DAG_ID='fruits_data_pipeline' AIRFLOW_CTX_TASK_ID='create_fruits_table' AIRFLOW_CTX_EXECUTION_DATE='2024-04-13T18:50:49.764352+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-13T18:50:49.764352+00:00'
[2024-04-13T18:50:56.964+0000] {sql.py:276} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS fruits (
            id SERIAL PRIMARY KEY,
            name VARCHAR(255),
            calories DECIMAL,
            fat DECIMAL,
            sugar DECIMAL,
            carbohydrates DECIMAL,
            protein DECIMAL
        )
    
[2024-04-13T18:50:57.000+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/eksplorasi.py", line 26, in create_fruits_table
    PostgresOperator(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    hook = self.get_db_hook()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 188, in get_db_hook
    return self._hook
  File "/usr/local/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 150, in _hook
    conn = BaseHook.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 82, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 514, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `postgres_default` isn't defined
[2024-04-13T18:50:57.015+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=fruits_data_pipeline, task_id=create_fruits_table, execution_date=20240413T185049, start_date=20240413T185056, end_date=20240413T185057
[2024-04-13T18:50:57.043+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 174 for task create_fruits_table (The conn_id `postgres_default` isn't defined; 588)
[2024-04-13T18:50:57.074+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-13T18:50:57.095+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SimpleHttpOperator. (Class `SimpleHttpOperator` is deprecated and will be removed in a future release. Please use `HttpOperator` instead.)
  result = cls.__new__(cls)

[2024-04-13T18:50:57.132+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
